vsts/build.yaml

```diff
@@ -13,9 +13,6 @@ jobs:
       versionSpec: '3.x'
       architecture: 'x64'
 
-  - script: 'python -m pip install --upgrade pip'
-    displayName: 'Update pip'
-
```

Comment:
Curious: why are we removing this?

Reply:
once we tart using env set up in any yaml that env set up also takes care of this

---
In reply to: [315397062](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r315397062) [](ancestors = 315397062)